pkgs/games/gargoyle/default.nix

```
@@ -19,14 +19,15 @@ let
 
 in
 
-stdenv.mkDerivation {
-  name = "gargoyle-2018-10-06";
+stdenv.mkDerivation rec {
+  name = "gargoyle";
```

```suggestion
  pname = "gargoyle";
```

Note that this should be `pname`, not `name`. `name` is defined in terms of `pname` and `version`.